code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -379,6 +379,9 @@ emp_act
 			return
 
 		var/obj/item/organ/external/affecting = get_organ(zone)
+		if(!affecting)
+			visible_message("\blue \The [O] misses [src] narrowly!")

```

Comment:
I'm not the one who actually implemented it; I just approved it. It was overwhelmingly liked (and not shocking; removing player agency at random times is terrible gameplay).